docs-kits/kits/Circularity_KIT/page-adoption-view.md

```diff
@@ -232,11 +232,13 @@ The schematic depiction above provides a visual representation of the connection
 
 #### 2. Order Data
 
-To clearly identify the order and relevant material, the order number (orderNumber) can be provided on an optional basis.
+To clearly identify the order and relevant material, the order number _(orderNumber)_ can be provided on an optional basis.
+
+#### B. Chain of Custody Models
 
-#### B. Accounting for Chain of Custody Models
+Disclaimer: For information regarding the calculation of a PCF on basis of the Chain of Custody Models please see the Catena-X [(PCF) Rulebook.](https://catena-x.net/de/aktuelles-termine/news-display/pcf-regelwerk-v20-bereit-fuer-ihr-feedback)
```

Comment:
Please link to the Catena-X standard library instead of an old link for consultation -  https://catena-x.net/de/standard-library

Reply:
Done. Created a dedicated Subpage

Reply:
Done! Updated link